### DISCUSSION TITLE
GitHub student pack approvval

### DISCUSSION THREAD

**Discussion by Parav04** [community/community#114735] [GitHub Education]
### Hi everyone,

Hi, I applied for the GitHub Student Pack on March 27th, and after receiving a confirmation message that said "Thanks for submitting," I haven't heard from them since. Could you please clarify if my request has been approved, or if I've been rejected? If it's approved, is it normal for it to take some time before my benefits arrive?

**Comment by HeyFang** [comment]
It should not take long, try reapplying or contacting github support

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
